include/odri_control_interface/calibration.hpp

```diff
@@ -48,12 +48,13 @@ class JointCalibrator
     double Kp_;
     double Kd_;
     double T_;
-    double dt_;
     double t_;
     bool go_to_zero_position_;
     int n_;
 
 public:
+    double dt;
```

Comment:
missing trailing underscore

Reply:
I think it's better in c++ to separate the internal data to the front end api.
If you need access to the dt simply add a getter.
If you need to set it, use a setter.

You can do both like those:
```
// write only
void dt(double in)
{
    dt_ = in;
}
// read only
const double& dt()
{
    return dt_;
}
// read/write
double& dt()
{
    return dt_;
}
```

Reply:
ps: pinocchio use the last 2, I think it's fairly elegant and prevent potential mistake removing the trailing underscore.

Reply:
Agreed, I will change it.